documentation/asciidoc/computers/raspberry-pi/eeprom-bootloader.adoc

```diff
@@ -481,6 +521,14 @@ PARTITION=2
 
 Default: 0
 
+[[PARTITION_WALK]]
+==== `PARTITION_WALK`
+This property is designed to improve the reliability of `A/B` boot schemes using `autoboot.txt` by searching for bootable partitions if the specified partition does not appear to be bootable. If `PARTITION_WALK=1` and the requested partition is not bootable and does not have a valid `autoboot.txt` then the bootloader will check each partition in turn (up to 8 and wrapping to 0) to see if it is bootable (contains `start.elf` or `config.txt` and a suitable device-tree on Pi5 or newer).
```

Comment:
I wonder if "(contains `start.elf` or `config.txt` and a suitable device-tree on Pi5 or newer)" would be clearer if written as "(contains `start.elf` on Pi 4, or `config.txt` and a suitable device-tree on Pi 5 or newer)" ? Possibly a question for @nathan-contino 

Also, "Pi5" should be written as "Pi 5" :wink:

Reply:
done